ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -340,6 +340,38 @@ function upgrade() {
     echo "Upgrading cluster to ${TARGET} gets started..."
 }
 
+# https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-73352
+function check_upgrade_recommend_when_upgrade_inprogress() {
+    if [[ "${SOURCE_MINOR_VERSION}" -eq "18" ]] || [[ "${SOURCE_MINOR_VERSION}" -eq "19" ]] ; then
```

Comment:
should be `TARGET_MINOR_VERSION`?

Reply:
Actually this is a OC feature, we need to check oc version greater then 4.17

Reply:
In this step, oc version is align with target version

Reply:
done